src/button-group/internal.tsx

```diff
@@ -39,7 +39,7 @@ const InternalButtonGroup = forwardRef(
     const containerObjectRef = useRef<HTMLDivElement>(null);
     const containerRef = useMergeRefs(containerObjectRef, __internalRootRef);
     const itemsRef = useRef<Record<string, ButtonProps.Ref | null>>({});
-    const [feedbackItemId, setFeedbackItemId] = useState<string | null>(null);
+    const [tooltip, setTooltip] = useState<null | { item: string; feedback: boolean }>(null);
```

Comment:
The central tooltip state guarantees there is only one tooltip shown at a time provided there is an ID check in the item.